kubernetes/crd/domain-crd.yaml

```diff
@@ -91,9 +136,18 @@ spec:
             domainHome:
               type: string
               description: The folder for the WebLogic Domain. Not required. Defaults
-                to /shared/domains/domains/domainUID if domainHomeInImage is false.
-                Defaults to /u01/oracle/user_projects/domains/ if domainHomeInImage
-                is true.
+                to /shared/domains/domains/domainUID if domainHomeSourceType is not
+                Image. Defaults to /u01/oracle/user_projects/domains/ if domainHomeSourceType
+                is Image.
+            domainHomeSourceType:
+              type: string
+              description: 'Domain home source type: Legal values: Image, PersistentVolume,
+                FromModel. Defaults to Image unless configuration.model is provided
+                and then defaults to FromModel.'
```

Comment:
Update this description if you agree to make this a static default instead of a dynamic default.

Reply:
Related question:  If a 'domainHomeInImage' is explicitly set (to true or false) AND domainHomeSourceType is also set, I assume the latter takes precedence?    And if domainHomeSourceType is not set, then domainHomeInImage takes precedence (if not set or true, it's domain-in-mage, if false it's domain-in-pv)...

Reply:
If you agree to make this aa static default, and my assumptions about handling the deprecated old field are correct, then the description should end with something like "Defaults to FromImage if the deprecated attribute modelInImage is either not set or explicitly set to 'true'.  Defaults to PersistentVolume if the deprecated attribute modelInImage is explicitly set to 'false'."